navigation/AppNavigation.js

```diff
@@ -71,10 +71,10 @@ const Tabs = () => {
 function AppNavigation(props) {
   return (
     <Stack.Navigator
-      initialRouteName="Sign In"
-      // screenOptions={{
-      //   headerShown: false,
-      // }}
+      initialRouteName="ParkedCarForm"
```

Comment:
This needs to be changed back please so it loads in the right place

Reply:
Sure